nixos/modules/services/x11/hardware/libinput.nix

```diff
@@ -156,6 +156,16 @@ let cfg = config.services.xserver.libinput;
           '';
       };
 
+      tappingButtonMap = mkOption {
+        type = types.nullOr (types.enum [ "lrm" "lmr" ]);
+        default = null;
+        description =
+          ''
+            Set   the   button   mapping   for   1/2/3-finger   taps  to  left/right/middle  or
+            left/middle/right, respectively.
+          '';
```

Comment:
```suggestion
        description = ''
          Set the button mapping for 1/2/3-finger taps to left/right/middle or left/middle/right, respectively.
        '';
```

Reply:
@SuperSandro2000 thank you for reviewing. I have committed your suggestion.

I also noticed that in the time the change had been sitting on my PC, everything else in the file had started using lib.mdDoc, so I have added that as well.